components/event-bus/cmd/event-bus-publish-knative/publisher/publisher.go

```diff
@@ -61,6 +63,13 @@ func (publisher *DefaultKnativePublisher) Publish(knativeLib *knative.KnativeLib
 	channel, err := knativeLib.GetChannel(*channelName, *namespace)
 	if err != nil {
 		log.Printf("cannot find the knative channel '%v' in namespace '%v'", *channelName, *namespace)
+		log.Println("incrementing total published messages counter")
+		log.Println("incrementing ignored messages counter")
```

Comment:
Please consider combining the two logs together to have the full context in one log line.

Reply:
Right, this was missed in the debugging phase